lib/frontend/admin/pages/auth/otp_email.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:animate_do/animate_do.dart';
+import 'package:email_otp/email_otp.dart';
+import 'package:flutter/material.dart';
+import 'dart:async';
+
+import 'package:flutter_verification_code/flutter_verification_code.dart';
+import 'package:rafay_portfolio/frontend/admin/view/dashboard.dart';
+
+class OtpEmailVerifyScreen extends StatefulWidget {
+  const OtpEmailVerifyScreen({super.key, required this.myauth});
+  final EmailOTP myauth;
+  @override
+  _OtpEmailVerifyScreenState createState() => _OtpEmailVerifyScreenState();
+}
+
+class _OtpEmailVerifyScreenState extends State<OtpEmailVerifyScreen> {
+  TextEditingController otp1Controller = TextEditingController();
+  TextEditingController otp2Controller = TextEditingController();
+  TextEditingController otp3Controller = TextEditingController();
+  TextEditingController otp4Controller = TextEditingController();
+  String _code = '';
+
+  int _currentIndex = 0;
+
+  @override
+  void initState() {
+    Timer.periodic(const Duration(seconds: 5), (timer) {
+      setState(() {
+        _currentIndex++;
+
+        if (_currentIndex == 3) {
+          _currentIndex = 0;
+        }
+      });
+    });
```

Comment:
There is no need to check for time and then cancel it, becase, this is only one time check and it will be canceled when the user will click on the button again.